build.gradle

```diff
@@ -119,3 +118,23 @@ tasks.withType(Test) {
     // Make sure it is UTF-8 for tests
     systemProperty "file.encoding", "UTF-8"
 }
+
+// Load signatory config
+['signing.keyId', 'signing.password', 'signing.secretKeyRingFile']
```

Comment:
what does this do? these properties don't seem to be used anywhere and the env vars are used instead

Reply:
I'ts a quirk, to configre the signatory you need these properties set on the project.
The environment variables e.g. `ORG_GRADLE_PROJECT_signing_secretKeyRingFile` set properties with an `_` not a `.` because AFAICT you can't include a dot in an env var name.
So `findProperty` searches for the values of the `_` version and this block sets it to the dotted version reqired by the signing extension.

Reply:
Thanks for explaining this - makes sense now

Reply:
As discussed perhaps d08a7c4 this makes things a little more clear and concise anyway